app/build.gradle

```diff
@@ -31,15 +29,19 @@ android {
             output.outputFileName = "$libName.${variant.name}.aar"
         }
     }
+
+    buildFeatures {
+        viewBinding true
+    }
```

Comment:
This was needed for the last kotlin version. It changes the ways you bind layouts in activities or fragments, but that's a change for later. The old way is still supported.